saleor/order/tests/test_fetch_order_prices_tax_app.py

```diff
@@ -0,0 +1,656 @@
+import json
+from decimal import Decimal
+from unittest.mock import Mock
+
+import pytest
+
+from ...discount import (
+    DiscountType,
+    DiscountValueType,
+    RewardType,
+    RewardValueType,
+    VoucherType,
+)
+from ...graphql.core.utils import to_global_id_or_none
+from ...tax import TaxableObjectDiscountType, TaxCalculationStrategy
+from ...webhook.event_types import WebhookEventSyncType
+from ...webhook.tests.subscription_webhooks.test_create_deliveries_for_taxes import (
+    TAXES_SUBSCRIPTION_QUERY,
+)
+from ...webhook.transport.synchronous.transport import (
+    create_delivery_for_subscription_sync_event,
+)
+from .. import OrderStatus
+from ..calculations import fetch_order_prices_if_expired
+from ..utils import create_order_discount_for_order
+
+
+@pytest.fixture
+def order_with_lines(order_with_lines):
```

Comment:
Same here. Create new fixture

Reply:
I want to override it. Please notice I don't use it as a test argument. I need to override it in order to change other fixtures behaviour. This way I avoid saving channel settings (extra db calls, time of execution) for each test.